apps/web/src/index.css

```
@@ -93,6 +93,44 @@
+
+  .page-break-node__label {
+    @apply text-xs font-medium text-notion-muted uppercase tracking-widest px-2 whitespace-nowrap;
+  }
+
+  /* ─── Table ─── */
+  .editor-table {
+    @apply w-full border-collapse my-4;
+  }
+
+  .editor-table-row {
+  }
```

Stylelint (17.4.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)